### PR TITLE
[CAS-895] Korean translation

### PR DESCRIPTION
**Pull request by bychkovdmitry**
### 🎯 Goal

Korean translations for the following strings:

```xml
<string name="stream_ui_message_list_pinned_message">Pinned by %s</string>
<string name="stream_ui_message_list_pinned_message_you">You</string>
<string name="stream_ui_message_list_pin_message">Pin to Conversation</string>
<string name="stream_ui_message_list_unpin_message">Unpin from Conversation</string>
<string name="stream_ui_message_list_error_pin_message">Failed to pin message</string>
<string name="stream_ui_message_list_error_unpin_message">Failed to unpin message</string>
<string name="stream_ui_pinned_message_list_empty_title">No pinned messages</string>
<string name="stream_ui_pinned_message_list_empty_description"><![CDATA[Long-press an important message and choose <b>Pin to conversation</b>]]></string>
<string name="stream_ui_pinned_message_list_results_error">An error occurred</string>
```

### ☑️ Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (docusaurus, tutorial, CMS (task created))
- [x] Reviewers added

### CODE REVIEW

File: stream-chat-android-ui-components/src/main/res/values-ko/strings_message_list.xml [hunk 1]
```diff
@@ -14,6 +14,8 @@
     <string name="stream_ui_message_list_attachment_uploading">%d/%dmb를 업로드중…</string>
     <string name="stream_ui_message_list_attachment_upload_complete">업로드 완료, 처리 중…</string>
     <string name="stream_ui_message_list_attachment_upload_progress">%s / %s</string>
+    <string name="stream_ui_message_list_pinned_message">%s에 의해 고정됨</string>
+    <string name="stream_ui_message_list_pinned_message_you">너</string>
```
Comment (EricSSartorius) [inline stream-chat-android-ui-components/src/main/res/values-ko/strings_message_list.xml:18]: 당신

File: stream-chat-android-ui-components/src/main/res/values-ko/strings_message_list.xml [hunk 2]
```diff
@@ -14,6 +14,8 @@
     <string name="stream_ui_message_list_attachment_uploading">%d/%dmb를 업로드중…</string>
     <string name="stream_ui_message_list_attachment_upload_complete">업로드 완료, 처리 중…</string>
     <string name="stream_ui_message_list_attachment_upload_progress">%s / %s</string>
+    <string name="stream_ui_message_list_pinned_message">%s에 의해 고정됨</string>
```
Comment (EricSSartorius) [inline stream-chat-android-ui-components/src/main/res/values-ko/strings_message_list.xml:17]: %s의 핀

File: stream-chat-android-ui-components/src/main/res/values-ko/strings_message_list.xml [hunk 3]
```diff
@@ -28,6 +30,8 @@
     <string name="stream_ui_message_list_copy_message">메시지를 복사하기</string>
     <string name="stream_ui_message_list_edit_message">메시지를 편집하기</string>
     <string name="stream_ui_message_list_flag_message">메시지를 플래그하기</string>
+    <string name="stream_ui_message_list_pin_message">대화에 고정</string>
```
Comment (EricSSartorius) [inline stream-chat-android-ui-components/src/main/res/values-ko/strings_message_list.xml:33]: 대화에 핀하기

File: stream-chat-android-ui-components/src/main/res/values-ko/strings_message_list.xml [hunk 4]
```diff
@@ -28,6 +30,8 @@
...
+    <string name="stream_ui_message_list_unpin_message">대화에서 고정 해제</string>
```
Comment (EricSSartorius) [inline stream-chat-android-ui-components/src/main/res/values-ko/strings_message_list.xml:34]: 대화에서 핀 해제

File: stream-chat-android-ui-components/src/main/res/values-ko/strings_message_list.xml [hunk 5]
```diff
@@ -57,4 +61,6 @@
     <string name="stream_ui_message_list_error_unmute_user">사용자의 음소거를 해제하지 못했습니다</string>
     <string name="stream_ui_message_list_error_block_user">사용자 차단 실패했습니다</string>
     <string name="stream_ui_message_list_error_flag_message">메시지에 플래그를 지정하지 못했습니다</string>
+    <string name="stream_ui_message_list_error_pin_message">메시지를 고정하지 못했습니다.</string>
```
Comment (EricSSartorius) [inline stream-chat-android-ui-components/src/main/res/values-ko/strings_message_list.xml:64]: 메시지를 핀하지 못했습니다

File: stream-chat-android-ui-components/src/main/res/values-ko/strings_message_list.xml [hunk 6]
```diff
@@ -57,4 +61,6 @@
...
+    <string name="stream_ui_message_list_error_unpin_message">메시지를 고정 해제하지 못했습니다.</string>
```
Comment (EricSSartorius) [inline stream-chat-android-ui-components/src/main/res/values-ko/strings_message_list.xml:65]: 메시지를 핀 해제하지 못했습니다

File: stream-chat-android-ui-components/src/main/res/values-ko/strings_pinned_message_list.xml [hunk 7]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="stream_ui_pinned_message_list_empty_title">고정된 메시지 없음</string>
```
Comment (EricSSartorius) [inline stream-chat-android-ui-components/src/main/res/values-ko/strings_pinned_message_list.xml:3]: 핀된 메시지 없음

File: stream-chat-android-ui-components/src/main/res/values-ko/strings_pinned_message_list.xml [hunk 8]
```diff
@@ -0,0 +1,6 @@
...
+    <string name="stream_ui_pinned_message_list_empty_description"><![CDATA[중요한 메시지를 길게 누르고 <b>대화에 고정</b>을 선택합니다.]]></string>
```
Comment (EricSSartorius) [inline stream-chat-android-ui-components/src/main/res/values-ko/strings_pinned_message_list.xml:4]: <string name="stream_ui_pinned_message_list_empty_description"><![CDATA[중요한 메시지를 길게 누르고 <b>대화에 핀하기</b>을 선택합니다.]]></string>

**Review by EricSSartorius — CHANGES_REQUESTED** [review]
Update korean translations

**Review by adasiewiczr — APPROVED** [review]

**Pull request merged**